cinema/urls.py

```diff
@@ -1 +1,22 @@
-# write urls here
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+
+from cinema.views import (
+    ActorViewSet,
+    CinemaHallViewSet,
+    GenreViewSet,
+    MovieSessionViewSet,
+    MovieViewSet,
+)
+
+router = DefaultRouter()
+router.register("actors", ActorViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("genres", GenreViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("movies", MovieViewSet)
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
```

Comment:
When basepath is empty no need to include, instead use:
urlpatterns = router.urls